File: PRAK204_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.py
import math

#data 
j = float("7")
t = float("10")

#hitung volume
volume = math.pi * j**2 * t

#hitung luas
luas = 2 * math.pi * j *(j+t)

#hitung keliling
keliling = 2 * math.pi * j 

#hasil
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: PRAK204_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.c
#include <stdio.h>

#define PI 3.14159265359

int main() {
    double jarijari, tinggi;
    double volume, luas, keliling;

    //Input Jari-jari&Tinggi
    printf("7");
    scanf("%lf",&jarijari);

    printf("10");
    scanf("%lf",&tinggi);

    //Hitungan
    volume = PI * jarijari * jarijari * tinggi;
    luas = 2 * PI * jarijari * (jarijari+tinggi);
    keliling = 2 * PI * jarijari;

    //Hasil
    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", keliling);

    return 0;
}